_faq/cep-and-its-importance.md

```diff
@@ -0,0 +1,25 @@
+---
+question: "What is a cEP and why is it so important to do a pre-GSoC analysis
+and design it as early as possible, for it to be finalized in the cEP?"
+---
+A **cEP (coala Enhancement Proposal)** is proposed to the **coala** community
+to discuss the new *major features* that the contributor intends to implement
+during the GSoC duration. The proposal is designed before the implementation of
+*major features* because the implementation usually happens from a technical
+perspective which may lead to a poorly designed content - developer and user-wise
+, so to avoid this, a cEP is designed at an earlier stage that can essentially
+save a lot of coding work and also makes sure that major features are properly
+documented.
+
+Experience tells us that the best GSoC students are the ones who heavily
+contribute to the project ideas' website, create projects from the issues, and
+expand on the existing ***.md*** files.
+
+So, if you are planning to participate in GSoC with coala, you should start
+writing a detailed project idea in our [projects](https://github.com/coala/projects)
+repository. Also, you must do enough research about the project in which you
+are interested in, so that you're able to express your ideas, timeline, etc
```

Comment:
`the project in which you are interested in`
"in" has become redundant here. You might want to consider using only one of the two.

Reply:
Will try to solve that redundancy.

Reply:
Yeah. Just keep either one of the two.